src/styles/blocks/card.scss

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+  font-family: $main-font;
+  color: $main-color;
+  font-weight: 400;
+
+  &:hover {
+    transform: scale(120%);
+    transition-duration: 300ms;
+    scroll-behavior: smooth;
+
+    &card__title{
+      color: #34568b;
+     }
+  }
+
+  &__img {
+    display: flex;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    padding: 0 16px 16px;
+  }
+
+  &__title {
+    margin: 40px 0 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $second-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-count {
+      font-family: $main-font;
+      text-align: right;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 16px;
+
+    &-title {
+      color: $second-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-amount {
+      text-align: right;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 40px;
+    color: $white-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    border-radius: 5px;
+    background-color: $main-color-btn;
+    transition: all 300ms;
```

Comment:
It is recommended to specify the transition properties instead of 'all' for better performance